docs/user_guide/tensorflow/model_serving.rst

```diff
@@ -24,73 +22,119 @@ To demonstrate this we provide an example notebook which is also included in the
 https://github.com/hopshadoop/hops-examples/blob/master/tensorflow/notebooks/Serving/train_and_export_model.ipynb
 
 In order to serve a TensorFlow model on Hopsworks, the .pb file and the variables folder should be placed in the Models dataset in your Hopsworks project. Inside the dataset, the folder structure should mirror what is expected by TensorFlow Serving.
-   
-.. _serving_structure.png: ../../_images/serving_structure.png
-.. figure:: ../../imgs/serving_structure.png
-   :alt: Expected file and folder structure for TensorFlow Serving
-   :target: `serving_structure.png`_
-   :align: center
-   :figclass: align-center
 
+.. code-block:: shell 
+
+    Models
+    └── mnist
+        ├── 1
+        │   ├── saved_model.pb
+        │   └── variables
+        │       ├── variables.data-00000-of-00001
+        │       └── variables.index
+        └── 2
+            ├── saved_model.pb
+            └── variables
+                ├── variables.data-00000-of-00001
+                └── variables.index
+
+TensorFlow serving expects the model directory (In this case *mnist*) to contain one or more sub-directories.
+The name of each sub-directory is a number representing the version of the model, the higher the version, the more recent the model.
+Inside each version directory TensorFlow serving expects a file named *saved_mode.pb*, which contains the graph of the model, and a directory called *variables* which contains the weights of the model.
 
 **Step 2.**
 
-The next step is to create a serving definition in the Hopsworks Model Serving service.
+To start serving your model, create a serving definition in the Hopsworks Model Serving service. Clik on the Model Serving service on the left panel (1) and then click on *Create new serving* (2).
```

Comment:
@SirOibaf `Select the Model Serving..` `and then select *Create new serving*`.

I prefer to write `select` instead of `click`. If users have a touch screen it might need to read `tap` instead of `click`. I'm not hard-minded on this one, just a suggestion.